### ISSUE TITLE
Issue loading sidekiq-scheduler/web

### ISSUE THREAD

**Issue by cbetta** [moove-it/sidekiq-scheduler#52]
Whatever I do, I keep getting this error: 

```
19:35:55 web.1     | [24973] ! Unable to load application: LoadError: cannot load such file -- sidekiq-scheduler/web
```

My load in `config/routes.rb` looks like this:

```rb
require 'admin_constraint'
require 'sidekiq/web'
require 'sidekiq-scheduler/web'

BattleHack::Application.routes.draw do
  ...
  mount Sidekiq::Web => '/admin/sidekiq', constraints: AdminConstraint.new
  ...
```

**Comment by cbetta** [comment]
BTW Gemfile contains:

```rb
gem "sidekiq"
gem "sidekiq-scheduler"
gem "sinatra", require: nil
```

**Comment by elpic** [comment]
The version is not pushed to rubygems yet. You can point to the github. In a day I will push the new version to rubygems.

**Comment by cbetta** [comment]
That explains :)

**Comment by elpic** [comment]
Gem released v1.2.1

**Comment by cbetta** [comment]
Tnx dude, great work!
